Tema-6/3/style.css

```diff
@@ -0,0 +1,125 @@
+*{
+    box-sizing: border-box;
+    
+}
+.first-title {
+    font-weight: 600;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    position: relative;
+    left: 10px;
+}
+
+.left {
+    display: flex;
+    margin: 10px 10px;
+    width: 70%;
+    border-bottom: 1px solid rgb(128, 128, 128, 0.4);
+    padding-bottom: 30px;
+}
+.left-side {
+    width: 250px;
+
+}
+.left p {
+    line-height: 18px;
+    color: rgba(80, 79, 79, 0.623);
+    font-weight: 100;
+}
+.left .image {
+    background-image:url("./china.png");    
+    width: 690px;
+    height: 430px;
+    margin-left: 20px;
+    margin-right: 10px;
+    padding: 1px;
+    background-size: cover;
+    border-left: 1px solid rgb(128, 128, 128, 0.4);
+}
+.right {
+    border-left: 1px solid rgb(128, 128, 128, 0.4);
+    margin: 0px 20px;
+    width: 27%;
+    position: absolute;
+    right: 0px;
+    top: 60px;
+    padding-left: 20px;
+    
+}
+
+
+.right .article-right {
+    padding:10px 10px;
+    border-bottom: 1px solid rgba(128, 128, 128,0.4);
+
+}
+.right  .paragraf {
+    margin-top: -17px;
+}
+.right .article-right1 {
+    margin-left: 8px;
+    border-bottom: 1px solid rgba(128, 128, 128,0.4);
+
+}
+.right .article-right1 h4 {
+    margin-top: 0px;
+}
+.article-right-no-border {
+    margin-left: 8px;
+
+}
+.article {
+    
+    width: 250px ;
+    height: 190px;
+    
+}
+.article1 {
+    border-bottom: 1px solid rgba(128, 128, 128,0.2);
+}
+.article1 h4 {
+    margin-top: 0px;
+}
+.right span {
+    position: relative;
+    bottom: 20px;
+    color: rgba(128, 128, 128, 2.6);
+    font-size: 10px;
+}
+span {
+    color: rgba(128, 128, 128, 2.6);
+    font-size: 10px;
+    font-weight: 100;
+}
+.bottom {
+    margin-top: 10px;
+    display: flex;
+    width: 68%;
+}
+.bottom .article-bottom1 {
+    margin: 0px 10px;
+    border-right: 1px solid rgb(128, 128, 128, 0.4);
+    margin-right: 20px;
+}
+.bottom .article-bottom2 {
```

Comment:
Pentru stilizarea claselor nu este necesar sa selectezi in baza elementului parinte. 
O clasa are un stil unic indiferent unde s-ar afla imediat cum ai declarat-o. Declaratia de tipul .parent child se face de obicei cand selectezi dupa elementul de html direct gen: .parent h1 sau .parent span etc
De obicei asa selectam un singur element care nu e de tip "container" ca sa zic asa adica sa nu fie div sau section etc. Dar un h1, h2m span, p sau genul asta de elemente este super ok.

Reply:
Ca sa iti dau un exemplu concret ca poate de deruteaza un pic explicatiile de mai sus:

<div class="container">
<p class="container-text">Lorem ipsum dolor <span>sancte</span> durum </p>
</div>

Poti aplica in felul asta:

.container{
font-size:20px;
color:white;
}

.container-text{
line-height:5px;
}

.container-text span{
color:red;
}

Tine cont ca daca vei mai avea un alt element "p" cu aceeasi denumire de clasa "container-text" cu un alt span in el si acel span va avea aceeasi stilizare ca cel anterior.